cfme/web_ui/paginator.py

```diff
@@ -123,6 +131,7 @@ def pages():
     # Yield while there are more pages
     while 'dimmed' not in next().get_attribute('class'):
         sel.click(next())
+        assert rec_offset() <= rec_total(), 'paginator goes beyond total number of items'
```

Comment:
The actual problem I'm trying to solve: https://bugzilla.redhat.com/show_bug.cgi?id=1205402